.github/workflows/conda-cpp-tests.yaml

```diff
@@ -69,18 +69,18 @@ jobs:
             pull-request:
               # amd64
               - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '11.8.0', LINUX_VER: 'rockylinux8', GPU: 'v100', DRIVER: 'earliest', DEPENDENCIES: 'oldest' }
-              - { ARCH: 'amd64', PY_VER: '3.12', CUDA_VER: '12.5.1', LINUX_VER: 'ubuntu22.04', GPU: 'v100', DRIVER: 'latest',   DEPENDENCIES: 'latest' }
+              - { ARCH: 'amd64', PY_VER: '3.12', CUDA_VER: '12.5.1', LINUX_VER: 'ubuntu24.04', GPU: 'v100', DRIVER: 'latest',   DEPENDENCIES: 'latest' }
               # arm64
               - { ARCH: 'arm64', PY_VER: '3.11', CUDA_VER: '12.0.1', LINUX_VER: 'ubuntu20.04', GPU: 'a100', DRIVER: 'latest',   DEPENDENCIES: 'latest' }
             nightly:
               # amd64
               - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '11.4.3', LINUX_VER: 'rockylinux8', GPU: 'v100', DRIVER: 'earliest', DEPENDENCIES: 'oldest' }
-              - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu22.04', GPU: 'v100', DRIVER: 'latest',   DEPENDENCIES: 'latest' }
+              - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu20.04', GPU: 'v100', DRIVER: 'latest',   DEPENDENCIES: 'latest' }
```

Comment:
Why has this been downgraded from 22.04 to 20.04?

Reply:
I rebalanced the OS version matrix a bit but this is open to change. As usual, our matrix should focus on the endpoints (oldest/newest). There is one job for Ubuntu 22.04 to provide additional coverage.

Would you prefer to make this 24.04?

Reply:
Actually, I'm sorry. We cannot make this 24.04 because Ubuntu 24.04 only supports CUDA 12.5+.

Our options are 22.04 or 20.04. Rocky is already covered on the line above so we should keep this on Ubuntu.

Reply:
For now, I'll revert to 22.04. I think that should be fine.

Reply:
Ergh. I read this again on my terminal (somehow it escaped me when reading the GitHub diff).

If we change this to 22.04, we will have _only_ ARM coverage for 20.04. We should test this endpoint ("earliest supported Ubuntu") on amd64 as well.